libvast/src/concept/parseable/vast/expression.cpp

```diff
@@ -20,6 +20,7 @@
 #include "vast/detail/assert.hpp"
 #include "vast/detail/string.hpp"
 #include "vast/expression.hpp"
+#include "vast/expression_visitors.hpp"
```

Comment:
The expander doesn't really belong in `expression_visitor.hpp`. Can you move it to this file, remove the include, and rename it to something more specific like `value_predicate_expander`?